tap_mssql/sync_strategies/full_table.py

```diff
@@ -64,29 +86,95 @@ def sync_table(mssql_conn, config, catalog_entry, state, columns, stream_version
 
     with mssql_conn.connect() as open_conn:
         LOGGER.info("Generating select_sql")
-        select_sql = common.generate_select_sql(catalog_entry, columns)
 
         params = {}
 
         if catalog_entry.tap_stream_id == "dbo-InputMetadata":
             prev_converter = modify_ouput_converter(open_conn)
 
-        common.sync_query(
-            open_conn,
-            catalog_entry,
-            state,
-            select_sql,
-            columns,
-            stream_version,
-            table_stream,
-            params,
-        )
+
+        columns.sort()
+        select_sql = common.fast_sync_generate_select_sql(catalog_entry, columns)
```

Comment:
I think we already discussed this, but again just to confirm - we want full syncs to always default to fast sync right? We don't even want the option to do a full sync without it?

Reply:
It's ok if that's what we want to do, but I think my preference is to provide the option to use either fast sync or the Singer spec for the full table refresh. That could look something like moving all of this logic into another method called `fastsync_table` and keeping the old logic in `sync_table`. From there in __init__.py in the method `do_sync_full_table` we could add a line for `if fastsync: fastsync_table` `else: sync_table`.

This way we could turn fast sync on on a table by table basis if we wanted.